fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2215,7 +2222,15 @@ public class Config extends ConfigBase {
      */
     @ConfField(mutable = true)
     public static boolean only_use_compute_node = false;
-
     @ConfField
     public static boolean enable_execute_script_on_frontend = true;
+    @ConfField(mutable = true)
+    public static boolean enable_stream_load_profile = false;
+
+    /**
+     * Only when the stream load time exceeds this value,
+     * the profile will be put into the profileManager
+     */
+    @ConfField(mutable = true)
+    public static long stream_load_profile_collect_millis = 10000; //10s
```

Comment:
`stream_load_profile_collect_threshold_seconds` I think seconds is enough. If the stream load is finished in one second, there is not necessary to collect profile.

Reply:
ok